spec/requests/admin/order_page_request_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Admin::Orders', type: :request do
+  let!(:admin) { create(:admin_user) }
```

Comment:
All of this 'let!' can be defined in context when admin is signed in.
It slow down your test for context `when admin not signed in`, you don't need to create it